tutorials/explanations/ai-explanations-tabular.ipynb

```diff
@@ -793,7 +793,9 @@
         "\n",
```

Comment:
<p>maybe simplify this slightly to something like</p><pre class="ql-syntax" spellcheck="false">attribution_vals = [x[0] for x in example['attributions_by_label'][0]['attributions'].values()]
</pre><p> </p><p>It might help to add a comment explaining these magical indexing <code>[0]</code>s. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ml-on-gcp/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Reply:
Good feedback, updated and added a comment